modules/installation-initializing.adoc

```diff
@@ -327,6 +327,13 @@ ifdef::vsphere,vmc[]
 +
 The installation program connects to your vCenter instance.
 ... Select the data center in your vCenter instance to connect to.
++
+[NOTE]
+====
+After you create the installation configuration file, you can modify the file to create an environment where you can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter.
+
+See xref:../../installing/modules/installation-vsphere-regions-zones.adoc[VMware vSphere region and zone enablement].
```

Comment:
Hi @bscott-rh . I've tried specifying a few different levels for this xref, but the link does not render. I wanted to point to the modules directly to avoid the ifdef scenario for teh various installation paths :grimacing: .

Not a good idea?